src/libs/blueprint/CMakeLists.txt

```diff
@@ -90,7 +90,7 @@ add_compiled_library(NAME conduit_blueprint
                      EXPORT conduit
                      HEADERS ${blueprint_headers} ${blueprint_c_headers}
                      SOURCES ${blueprint_sources} ${blueprint_c_sources} ${blueprint_fortran_sources}
-                     DEPENDS_ON conduit
+                     DEPENDS_ON conduit mpi
```

Comment:
quick note: we don't want the base blueprint lib to depend on mpi, only the blueprint_mpi lib

Reply:
Good catch--I temporarily had some MPI code in conduit_blueprint_mesh before conduit_blueprint_mpi_mesh existed, but this was not intended to be merged.